.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 services:
   - rabbitmq
 env:
+  global:
+  - GRADLE_OPTS="-Xmx512m"
```

Comment:
It failed once in Travis with some "killed" message, my guess is that main gradle process used too much memory.